gallery/src/components/NxTag/NxTagPage.tsx

```diff
@@ -61,6 +61,21 @@ const NxTagPage = () =>
             <NxTableCell>Default</NxTableCell>
             <NxTableCell>Details</NxTableCell>
           </NxTableRow>
+          <NxTableRow>
+            <NxTableCell>HTML div Attributes</NxTableCell>
+            <NxTableCell>
+              <a target="_blank"
+                 rel="noopener"
+                 href="https://developer.mozilla.org/es/docs/Web/HTML/Elemento/div">
+                HTML div Attributes
+              </a>
+            </NxTableCell>
+            <NxTableCell>No</NxTableCell>
+            <NxTableCell>
+              NxDropdown supports any html attribute that's normally supported by
```

Comment:
`NxDropdown` eh?

Reply:
> `NxDropdown` eh?

Busted on the copy-pasta: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/fe3ca665091980b25a1483c467a6c57979f19356